build-parent/pom.xml

```diff
@@ -992,6 +1019,33 @@ To suppress enforce-helm-version rule, you must add following plugin to the root
                                 <useDevRepository>true</useDevRepository>
                             </configuration>
                         </plugin>
+                        <plugin>
+                            <groupId>org.technologybrewery.fabric8</groupId>
+                            <artifactId>docker-maven-plugin</artifactId>
+                            <version>${version.fabric8.docker.maven.plugin}</version>
+                            <executions>
+                                <execution>
+                                    <id>default-build</id>
+                                    <phase>package</phase>
+                                    <goals>
+                                        <goal>build</goal>
+                                    </goals>
+                                    <configuration>
```

Comment:
I: When the `ci` profile is activated, do NOT build. We can improve performance by running both the `arm64` and `amd64` platform builds simultaneously.